core/src/test/java/org/elasticsearch/index/mapper/multifield/MultiFieldTests.java

```diff
@@ -255,125 +255,6 @@ public void testConvertMultiFieldNoDefaultField() throws Exception {
         assertThat(docMapper.mappers().getMapper("age.stored").fieldType().tokenized(), equalTo(false));
     }
 
-    public void testConvertMultiFieldGeoPoint() throws Exception {
-        Version version = VersionUtils.randomVersionBetween(random(), Version.V_1_0_0, Version.CURRENT);
-        Settings settings = Settings.settingsBuilder().put(IndexMetaData.SETTING_VERSION_CREATED, version).build();
-        boolean indexCreatedBefore22 = version.before(Version.V_2_2_0);
```

Comment:
this is because the test passes an array to a field that is mapped as a string and today the behaviour is horrible: the string only sees a "]" so an error is much better imo